packages/docusaurus-theme-classic/src/theme/Layout/styles.css

```diff
@@ -15,7 +15,7 @@ body {
   transition: var(--ifm-transition-fast) ease color;
 }
 
-body > div {
+body > div:first-of-type {
```

Comment:
Should we do `body > #__docusaurus`? There's no guarantee someone will not inject more `<div>` at the start of the `<body>`